Horion/Module/Modules/CrystalAura.cpp

```diff
@@ -53,13 +54,24 @@ bool CanPlaceC(vec3_ti* pos) {
 	g_Data.forEachEntity([](C_Entity* ent, bool b) {
 		if (!space)
 			return;
-		if (ent->aabb.intersects(*new AABB(_pos, _pos.add(1.f))))
+		if (ent->aabb.intersects(*new AABB(_pos, 1.f, 1.f)))
```

Comment:
AABB::AABB': no overloaded function takes 3 arguments

Reply:
it can handle 4 tho?????

Reply:
I made a custom overload then realized that u can just use add, but I was commiting to 2 different branches so I probably overwrote it.
Gonna fix it later

Reply:
Fixed it